js/blocks/controls/range.js

```diff
@@ -9,10 +9,12 @@ const BlockLabRangeControl = ( props, field, block ) => {
 			afterIcon={field.afterIcon}
 			label={field.label}
 			help={field.help}
-			value={attr[ field.name ] || field.default}
+			value={ ( attr[ field.name ] || '0' == attr[ field.name ] ) ? attr[ field.name ] : field.default }
 			onChange={rangeControl => {
-				attr[ field.name ] = rangeControl
-				setAttributes( attr )
+				if ( undefined !== rangeControl ) {
```

Comment:
What do you think about removing this conditional?
```js
if ( undefined !== rangeControl ) {
```
With that conditional, it looks like it's not possible to:
1. highlight and delete the value
2. backspace when there's only 1 digit

![values-here-and-here](https://user-images.githubusercontent.com/4063887/59648239-aca65700-917e-11e9-8c15-50fbea2c19ba.gif)

Reply:
This should be addressed in https://github.com/getblocklab/block-lab/pull/341/commits/d4baa80afd17569bd11f1a5d93def8714718e309 

Thanks for the feedback.

Reply:
Nice, https://github.com/getblocklab/block-lab/commit/d4baa80afd17569bd11f1a5d93def8714718e309 looks good.